Examples/Nodify.Playground/EditorInputMode.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System.Windows.Input;
+
+namespace Nodify.Playground
+{
+    public enum EditorInputMode
+    {
+        Default,
+        PanOnly,
+        SelectOnly
+    }
+
+    public enum EditorInputMappings
+    {
+        Default,
+        Blender3D
+    }
+
+    public static class EditorInputModeExtensions
+    {
+        public static void Apply(this EditorGestures mappings, EditorInputMode inputMode)
+        {
+            mappings.Apply(PlaygroundSettings.Instance.EditorInputMappings.ToInputMappings());
+
+            switch (inputMode)
+            {
+                case EditorInputMode.PanOnly:
+                    mappings.Editor.Selection.Apply(EditorGestures.SelectionGestures.None);
+                    mappings.ItemContainer.Selection.Apply(EditorGestures.SelectionGestures.None);
+                    mappings.ItemContainer.Drag.Value = MultiGesture.None;
+                    mappings.Connector.Connect.Value = MultiGesture.None;
+                    break;
+                case EditorInputMode.SelectOnly:
+                    mappings.Editor.Pan.Value = MultiGesture.None;
+                    mappings.ItemContainer.Drag.Value = MultiGesture.None;
+                    mappings.Connector.Connect.Value = MultiGesture.None;
+                    break;
+                case EditorInputMode.Default:
+                    break;
+            }
+        }
+
+        public static void Apply(this EditorGestures value, EditorInputMappings mappings)
+        {
+            var newMappings = mappings.ToInputMappings();
+            value.Apply(newMappings);
+        }
+
+        public static EditorGestures ToInputMappings(this EditorInputMappings mappings)
+        {
+            return mappings switch
+            {
+                EditorInputMappings.Blender3D => new Blender3DInputMappings(),
+                _ => new EditorGestures()
+            };
+        }
+    }
+
+    public class Blender3DInputMappings : EditorGestures
```

Comment:
I suggest using a different name so that others will understand the purpose of this mapping. Maybe `MoveInputMapping` or `PanInputMapping`...?

Reply:
Renamed it to CustomGesturesMappings